src/Connector.AzureDataLake/Connector/AzureDataLakeClient.cs

```diff
@@ -22,6 +22,12 @@ public async Task<DataLakeDirectoryClient> EnsureDataLakeDirectoryExist(AzureDat
         }
 
         public async Task SaveData(AzureDataLakeConnectorJobData configuration, string content, string fileName)
+        {
+            using var stream = new MemoryStream(System.Text.Encoding.UTF8.GetBytes(content));
+            await SaveData(configuration, stream, fileName);
+        }
+
+        public async Task SaveData(AzureDataLakeConnectorJobData configuration, Stream stream, string fileName)
```

Comment:
This overload is only used by the above SaveData. I think you used it before creating the ExportDataJob